From 40f317b178bb38aeaaf47e5e1614a21aafd75d4a Mon Sep 17 00:00:00 2001
From: Jerin Jacob <jerin.jacob@caviumnetworks.com>
Date: Tue, 28 Feb 2017 08:54:32 +0000
Subject: [PATCH] hack to use cavium pci devices

---
 usertools/dpdk-devbind.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/usertools/dpdk-devbind.py b/usertools/dpdk-devbind.py
index 1b9c651..21cb1ce 100755
--- a/usertools/dpdk-devbind.py
+++ b/usertools/dpdk-devbind.py
@@ -40,7 +40,7 @@
 
 # The PCI base class for NETWORK devices
 NETWORK_BASE_CLASS = "02"
-CRYPTO_BASE_CLASS = "0b"
+CRYPTO_BASE_CLASS = "08"
 
 # global dict ethernet devices present. Dictionary indexed by PCI address.
 # Each device within this is itself a dictionary of device properties
-- 
1.9.1

